src/WormholeOracleAuth.sol

```diff
@@ -38,8 +38,17 @@ contract WormholeOracleAuth {
     event SignersAdded(address[] signers);
     event SignersRemoved(address[] signers);
 
+    // --- Errors ---
+    error NotAuthorized(address sender, uint256 wards);
```

Comment:
Similar to Join. `NotAuthorized` should be just the sender.

Reply:
done in 0172295